ci/build_cpp.sh

```diff
@@ -14,3 +18,20 @@ rapids-logger "Begin cpp build"
 rapids-mamba-retry mambabuild conda/recipes/libraft
 
 rapids-upload-conda-to-s3 cpp
+
+
+rapids-logger "ls conda-bld"
+
+ls -al /opt/conda/conda-bld/
+
+rapids-logger "ls conda-bld/*libraft*/"
```

Comment:
```suggestion
rapids-logger "Upload ninja logs"
```

All this fluff can be removed in the future